zcash_client_backend/src/decrypt.rs

```diff
@@ -49,21 +52,29 @@ pub fn decrypt_transaction(
         };
 
         for (account, (ivk, ovk)) in vks.iter().enumerate() {
-            let ((note, to, memo), outgoing) =
-                match try_sapling_note_decryption(ivk, &epk, &output.cmu, &output.enc_ciphertext) {
-                    Some(ret) => (ret, false),
-                    None => match try_sapling_output_recovery(
-                        ovk,
-                        &output.cv,
-                        &output.cmu,
-                        &epk,
-                        &output.enc_ciphertext,
-                        &output.out_ciphertext,
-                    ) {
-                        Some(ret) => (ret, true),
-                        None => continue,
-                    },
-                };
+            let ((note, to, memo), outgoing) = match try_sapling_note_decryption(
+                parameters,
+                height,
```

Comment:
`height` and `parameters` are in a different order here to in `decrypt_transaction`. (Removing `parameters` for now would make that moot.)

Reply:
`parameters` has been removed.